### PR TITLE
add the count character method

### PR DESCRIPTION
**Pull request by virginiaSanabria**

### CODE REVIEW

File: src/main/java/org/minions/devfund/Library.java [hunk 1]
```diff
@@ -1,16 +0,0 @@
-package org.minions.devfund;
-
-/**
- * This Java source file was auto generated by running 'gradle buildInit --type java-library'.
- */
-public class Library {
```
Comment (carledriss) [inline src/main/java/org/minions/devfund/Library.java]: Why are you deleting this code ?
Only commit your changes.

File: src/test/java/org/minions/devfund/TaskOneTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import org.junit.Test;
+
+public class TaskOneTest {
+
+    /**
+     * Verify some TaskOne method.
+     */
+    @Test
+    public void testOne() {
+        TaskOne taskOne = new TaskOne();
```
Comment (carledriss) [inline src/test/java/org/minions/devfund/TaskOneTest.java]: You are repeating this line of code in all your tests.
Please move it to a Before method.

File: src/test/java/org/minions/devfund/TaskOneTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        int result = taskOne.countChar("fizzbuzz",'z');
+        Assert.assertEquals(4, result);
```
Comment (carledriss) [inline src/test/java/org/minions/devfund/TaskOneTest.java]: Consider using a static import for Assert

File: src/main/java/org/minions/devfund/Library.java (hunk 1 above)
Comment (virginiaSanabria) [reply src/main/java/org/minions/devfund/Library.java]: done

File: src/test/java/org/minions/devfund/TaskOneTest.java (hunk 2 above)
Comment (virginiaSanabria) [reply src/test/java/org/minions/devfund/TaskOneTest.java]: done

File: src/test/java/org/minions/devfund/TaskOneTest.java (hunk 3 above)
Comment (virginiaSanabria) [reply src/test/java/org/minions/devfund/TaskOneTest.java]: what do you mean ?

**Review by carledriss — APPROVED** [review]
LGTM! 👍

**Pull request merged**